internal/model/models.go

```diff
@@ -1,15 +1,13 @@
 package model
 
 import (
-	"time"
-
 	"github.com/G-Research/yunikorn-core/pkg/webservice/dao"
 )
 
-type ApplicationDAOInfo struct {
-	CreatedAt              time.Time `json:"createdAt"`
-	QueueID                string    `json:"queueId"`
-	dao.ApplicationDAOInfo `json:",inline"`
+type ModelMetadata struct {
+	ID        string `json:"id"`
+	CreatedAt int64  `json:"createdAt"`
```

Comment:
If it isn't a `time.Time`, can we name it `CreatedAtNano` or `CreatedAtMilli` so the precision is not ambiguous.

Reply:
+1 on the renaming suggestion.